component.json

```diff
@@ -1,6 +1,9 @@
 {
   "title": "XML",
-  "description": "Component to work with XML files",
```

Comment:
is there any specific reason why the component description is removed?
![image](https://user-images.githubusercontent.com/36419533/79637965-b0827b80-818b-11ea-86c1-6143dfea2e13.png)

Reply:
I'll change that back. I guess that component level descriptions still appear even though action/trigger level descriptions are not longer rendered. Asking for follow up: https://elasticio.slack.com/archives/C6G1E4WB1/p1587374522293600

Reply:
Platform issue: https://github.com/elasticio/elasticio/issues/3816